lib/valkyrie/specs/shared_specs/persister.rb

```diff
@@ -410,27 +412,64 @@ class MyLockingResource < Valkyrie::Resource
   end
 
   context 'ordered properties' do
-    before do
-      raise 'persister must be set with `let(:persister)`' unless defined? persister
-      class OrderedCustomResource < Valkyrie::Resource
-        include Valkyrie::Resource::AccessControls
-        attribute :id, Valkyrie::Types::ID.optional
-        attribute :authors, Valkyrie::Types::Array.optional.meta(ordered: true)
-      end
+    it "orders string values and returns them in the appropriate order" do
+      validate_order ["a", "b", "a"]
     end
-    after do
-      Object.send(:remove_const, :OrderedCustomResource)
+
+    it "orders boolean values and returns them in the appropriate order" do
+      validate_order [true, false, true]
+    end
+
+    it "orders integer values and returns them in the appropriate order" do
+      validate_order [1, 2, 1]
+    end
+
+    it "orders date values and returns them in the appropriate order" do
+      now = Time.now.round(3).utc
+      validate_order [now, now - 3.hours, now - 1.hour]
+    end
+
+    it "orders URIs and returns them in the appropriate order" do
+      uri1 = RDF::URI("http://example.com/foo")
+      uri2 = RDF::URI("http://example.com/bar")
+      uri3 = RDF::URI("http://example.com/baz")
+      validate_order [uri1, uri2, uri3]
+    end
+
+    it "orders IDs and returns them in the appropriate order" do
+      page1 = persister.save(resource: resource_class.new(authors: ["Page 1"]))
+      page2 = persister.save(resource: resource_class.new(authors: ["Page 2"]))
+      page3 = persister.save(resource: resource_class.new(authors: ["Page 3"]))
+      validate_order [page1.id, page2.id, page3.id]
+    end
+
+    it "orders floating point values and returns them in the appropriate order" do
+      validate_order [1.123, 2.222, 1.123]
     end
-    subject { persister }
-    let(:resource_class) { OrderedCustomResource }
-    let(:resource) { resource_class.new }
-    it "saves ordered properties and returns them in the appropriate order" do
-      resource.authors = ["a", "b", "a"]
+
+    # does not work in fedora
+    xit "orders nested objects" do
```

Comment:
@escowles can you use a `pending` here with some info such as "does not work yet," or other explanation.